src/vs/workbench/services/configurationResolver/browser/configurationResolverService.ts

```diff
@@ -184,7 +182,7 @@ export abstract class BaseConfigurationResolverService extends AbstractVariableR
 				case 'command':
 					// use the name as a command ID #12735
 					const commandId = (variableToCommandMap ? variableToCommandMap[name] : undefined) || name;
-					result = await this.commandService.executeCommand(commandId, configuration);
+					result = await this.commandService.executeCommand(commandId, object);
```

Comment:
The object passed into executeCommand now has all the previously evaluated replacements inserted.